addons/escoria-core/game/core-scripts/esc/commands/anim.gd

```diff
@@ -56,9 +55,7 @@ func run(command_params: Array) -> int:
 
 # Function called when the command is interrupted.
 func interrupt():
-	escoria.logger.report_warnings(
-		get_command_name(),
-		[
-			"Interrupt() function not implemented"
-		]
+	escoria.logger.warn(
+		self,
+		"[%s] Interrupt() function not implemented." % get_command_name()
```

Comment:
```suggestion
		"[%s] interrupt() method not implemented." % get_command_name()
```

This might be a bit picky, but it speaks to accuracy:

- "interrupt" should be lowercase since the method name itself is case sensitive and it is "interrupt()", not "Interrupt()"
- Technically "interrupt" is a method and not a function. (Actually the more I think about this, the less I care as it's probably a little too pedantic...Feel free to ignore this part of the suggestion.)

This of course would need to be changed across all commands that have this "not implemented" message.